### ISSUE TITLE
No error displayed when spreading an object in a component

### ISSUE THREAD

**Issue by AliceLeRaptor** [sveltejs/eslint-plugin-svelte#1072]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

9.20.1

### What version of `eslint-plugin-svelte` are you using?

2.46.1

### What did you do?

<details>
<summary>Configuration</summary>

OS: Ubuntu 22.04.5 LTS
VSCode: 1.96.4
VSCode ESLint plugin `dbaeumer.vscode-eslint` version : 3.0.10
VSCode Svelte plugin `svelte.svelte-vscode` version : 109.5.2

Packages used :

```
"peerDependencies": {
	"svelte": "^5.0.0"
},
"devDependencies": {
	"@eslint/compat": "^1.2.6",
	"@eslint/js": "^9.20.0",
	"@sveltejs/adapter-auto": "^4.0.0",
	"@sveltejs/kit": "^2.17.1",
	"@sveltejs/package": "^2.3.10",
	"@sveltejs/vite-plugin-svelte": "^5.0.3",
	"eslint": "^9.20.1",
	"eslint-config-prettier": "^10.0.1",
	"eslint-plugin-svelte": "^2.46.1",
	"globals": "^15.14.0",
	"prettier": "^3.5.0",
	"prettier-plugin-svelte": "^3.3.3",
	"publint": "^0.3.4",
	"svelte": "^5.19.10",
	"svelte-check": "^4.1.4",
	"typescript": "^5.7.3",
	"typescript-eslint": "^8.24.0",
	"vite": "^6.1.0"
},

```
</details>

Hello and thank you for taking the time to read this issue.

Here are simple reproduction steps :

I created a new Svelte library project using the command `npx sv create`. I chose to use Typescript, ESLint and Prettier with pnpm.

I open the project in VSCode, then run npm-check-updates with `ncu -u` to update packages to their latest version (see Configuration for installed versions). I use `pnpm i` to install updated packages.

I create a Button.svelte component in`src/lib` with this content : 

```svelte
<script lang="ts">
    let props = $props();
</script>

<button {...props}></button>
```

### What did you expect to happen?

I expected to see the following error occur, since the `<button>` is missing a contain text : 
```
Buttons and links should either contain text or have an `aria-label` or `aria-labelledby` attribute
https://svelte.dev/e/a11y_consider_explicit_labelsvelte(a11y_consider_explicit_label)
```

### What actually happened?

There is no error displayed.

But I noticed that removing the spread `props` makes the error appear. I tested with few other spread use cases (in reproduction example) and the error is always missing both in VSCode and when using `npx eslint`.

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/AliceLeRaptor/new-project-eslint

**Comment by baseballyama** [comment]
I can see the error.

<img width="841" alt="Image" src="https://github.com/user-attachments/assets/2971af28-881b-4a92-9045-32d43d23c079" />

**Comment by AliceLeRaptor** [comment]
Yes in line 10 there is no spread object, there is indeed an error.

If you check the Button.svelte file there are other buttons with a spread object, also without text, that should display an error but do not.

**Comment by baseballyama** [comment]
This is a warning from the Svelte compiler. Svelte compiler does not do advanced type checking, so it doesn’t warn when using the spread syntax. The `svelte/valid-compile` rule just passes on the Svelte compiler warnings, so nothing can be done here. Therefore, I will close this issue.
